packages/api/route.go

```diff
@@ -81,6 +78,14 @@ func Route(route *hr.Router) {
 	post(`test/:name`, ``, getTest)
 	post(`content`, `template:string`, jsonContent)
 
+	if !*conf.IsVDEMode {
```

Comment:
why ecosystemparam route is allowed in VDE mode? The same thing about systemparams route

Reply:
be more attentive please. Should it work in vde mode?
get(`ecosystems`, ``, authWallet, ecosystems)